widgets/sidebar/sidebar.tsx

```diff
@@ -85,11 +89,14 @@ export const Sidebar = () => {
         Log Out
       </Typography.Regular14>
 
-      <DEPRECATED_Modal
-        onClose={() => setOpen(false)}
+      <ConfirmModal
```

Comment:
You should remove the unused CSS after replacing the hardcoded modal window with `ConfirmModal`.